CHANGES.md

```diff
@@ -1,3 +1,10 @@
+## [v4.7.0]
```

Comment:
Please don't add a version number or publish date in your PR (see [our contributing guidelines](https://github.com/ReactTraining/history/blob/master/CONTRIBUTING.md)).

Reply:
done